ecommerce-ios/Extensions/View+NavigationBar.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  View+NavigationBar.swift
+//  ecommerce-ios
+//
+//  Created by Su T. Nguyen on 13/05/2021.
+//
+
+import SwiftUI
+
+public extension View {
+
+    func navigationBarLargeTitleItems<T>(trailing: T) -> some View where T: View {
+        overlay(NavigationBarLargeTitleItems(trailing: trailing).frame(width: 0, height: 0))
```

Comment:
Let's add line break here and for each inlines for easy read. What do you think?
```suggestion
        overlay(NavigationBarLargeTitleItems(trailing: trailing)
            .frame(width: 0, height: 0))
```

Reply:
ye, totally agree 👍